pkg/auth/event/user_update.go

```diff
@@ -104,10 +104,11 @@ func (event UserUpdateEvent) WithMutationsApplied(mutations Mutations) UserAware
 	}
 	if mutations.VerifyInfo != nil {
 		newEvent.VerifyInfo = mutations.VerifyInfo
-		// IsVerified will be updated by mutator, if it is not also mutated
+		// IsComputedVerified will be updated by mutator
 	}
-	if mutations.IsVerified != nil {
-		newEvent.IsVerified = mutations.IsVerified
+	if mutations.IsComputedVerified != nil && mutations.IsManuallyVerified != nil {
```

Comment:
`mutations.IsComputedVerified` will be non nil, if `VerifyInfo` or `IsManuallyVerified` change. So If user change `VerifyInfo` info only and `IsManuallyVerified ` will be nil. And this checking is incorrect that we should have `IsVerified ` in the new event for this case, right?